pkg/kubelet/sysctl/namespace.go

```diff
@@ -43,6 +43,7 @@ var prefixNamespaces = map[string]Namespace{
 	"kernel.msg": ipcNamespace,
 	"fs.mqueue.": ipcNamespace,
 	"net.":       netNamespace,
+	"fs.file-max": ipcNamespace,
```

Comment:
@maguie i don't see any kernel docs that this parameter can be namespaced? also need to update tests if this is a valid change.

Reply:
This should be unsafe sysctl. It will change the node `fs.file-max`.
For instance, if one pod set `fs.file-max` to a very low number like 1-100, the node will become unhealthy.